package.json

```diff
@@ -73,19 +73,19 @@
 		"express": "^4"
 	},
 	"devDependencies": {
-		"@jest/globals": "^27.4.2",
+		"@jest/globals": "^27.4.6",
 		"@types/express": "^4.17.13",
-		"@types/jest": "^27.0.3",
-		"@types/node": "^16.11.17",
+		"@types/jest": "^27.4.0",
+		"@types/node": "^17.0.8",
```

Comment:
I would not use a non LTS node version here

Reply:
Arghhh that's the second time that's happened - I bump all the deps and this gets bumped too 🤦🏾 Maybe I should pin that to 16.x instead, until the next LTS version.